DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/vlan/test_vlan_untagged_ports.py

```diff
@@ -0,0 +1,129 @@
+import asyncio
+import pytest
+
+from dent_os_testbed.utils.test_utils.tgen_utils import (tgen_utils_get_dent_devices_with_tgen,
+                                                         tgen_utils_setup_streams, tgen_utils_start_traffic,
+                                                         tgen_utils_stop_traffic, tgen_utils_get_traffic_stats,
+                                                         tgen_utils_get_loss, tgen_utils_dev_groups_from_config,
+                                                         tgen_utils_traffic_generator_connect, )
+from dent_os_testbed.utils.test_utils.br_utils import (configure_bridge_setup, configure_vlan_setup,
+                                                       get_traffic_port_vlan_mapping)
+
+
+port_map = ({"port": 0, "settings": [{"vlan": 2, "untagged": True, "pvid": True}]},
+            {"port": 1, "settings": [{"vlan": 2, "untagged": True, "pvid": True}]},
+            {"port": 2, "settings": [{"vlan": 2, "untagged": True, "pvid": True}]},
+            {"port": 3, "settings": [{"vlan": 2, "untagged": True, "pvid": True}]})
+
+
+@pytest.mark.suite_functional_vlan
+@pytest.mark.asyncio
+@pytest.mark.usefixtures("cleanup_bridges", "cleanup_tgen")
+@pytest.mark.parametrize("traffic_type", ["broadcast", "multicast", "unicast"])
+async def test_vlan_untagged_ports_with_(testbed, traffic_type):
+    """
+    Test Name: VLAN untagged ports
```

Comment:
I would use an f-string so the test description will change with different arguments:
```suggestion
    f"""
    Test Name: VLAN untagged ports {traffic_type}
```

Reply:
i tried once< and for some reason it did not changed for me. Stayed like this ` VLAN untagged ports {traffic_type}`

Reply:
okay, this is not supported by pytest

Reply:
https://github.com/python/cpython/commit/d4e89287b397c7382c12d3f3d9fd901fd8243b3c

Reply:
If I'm not mistaken you can have variables in docstring and then in the function do:
```
test_vlan_untagged_ports_with_.__doc__ = test_vlan_untagged_ports_with_.__doc__.format(traffic_type)
```

Reply:
should we add it? or the output like below is fine 
```
415.65s call     functional/vlan/test_vlan_all_supported_modes.py::test_vlan_tagged_ports_with_[multicast]
388.40s call     functional/vlan/test_vlan_all_supported_modes.py::test_vlan_tagged_ports_with_[multicast]
0.01s setup    functional/vlan/test_vlan_all_supported_modes.py::test_vlan_tagged_ports_with_[multicast]
```